src/main/java/io/github/amerebagatelle/fabricskyboxes/Test.java

```diff
@@ -25,7 +26,7 @@
 public class Test implements PreLaunchEntrypoint {
     @Override
     public void onPreLaunch() {
-        if (!FabricLoader.getInstance().isDevelopmentEnvironment()) {
+        if (!FabricLoader.getInstance().isDevelopmentEnvironment() || System.getProperty("fabricskyboxes.runTests") == null || !System.getProperty("fabricskyboxes.runTests").equals("true")) {
```

Comment:
The one thing before I do merge, remove this.  I plan to move the tests stuff to the testmod fully.